interface/src/app/sample_annotation/annotation.spec.js

```diff
@@ -0,0 +1,68 @@
+describe('sample_annotation', function() {
+  var $httpBackend, $log, SampleMocks;
+
+  beforeEach(module('adage.sampleAnnotation'));
+  beforeEach(module('adage.mocks.sample'));
+  beforeEach(inject(function(_$httpBackend_, _$log_, _SampleMocks_) {
+    $httpBackend = _$httpBackend_;
+    $log = _$log_;
+    SampleMocks = _SampleMocks_;
+  }));
+
+  // Ensure no outstanding expectation or request at the end.
+  afterEach(function() {
+    $httpBackend.verifyNoOutstandingExpectation();
+    $httpBackend.verifyNoOutstandingRequest();
+    // very useful $log capture information from:
+    // http://www.jvandemo.com/how-to-access-angular-log-debug-messages-from-within-karma/
+    if ($log.debug.logs.length > 0) {
+      console.log($log.debug.logs);
+    }
+  });
+
+  describe('SampleAnnotationCtrl', function() {
+    var SampleAnnotationCtrl, $location, $scope, mockStateParams, $log;
+
+    beforeEach(inject(
+      function($controller, _$location_, _$q_, _$log_, $rootScope) {
+        $location = _$location_;
+        $log = _$log_;
+        $scope = $rootScope.$new();
+        mockStateParams = {samples: '1'};
+        SampleAnnotationCtrl = $controller('SampleAnnotationCtrl', {
+          $location: $location,
+          $scope: $scope,
+          $log: $log,
+          $stateParams: mockStateParams
+        });
+      }
+    ));
+
+    it('should pass a dummy test', inject(function() {
+      $httpBackend.expectGET(
+        '/api/v0/sample?id__in=1&limit=0'
+      ).respond(SampleMocks.sample1Paginated);
+      $httpBackend.flush();
+
+      expect(SampleAnnotationCtrl).toBeTruthy();
+    }));
+
+    it('should render the SampleAnnotationCtrl for a sample', inject(
```

Comment:
Since the injected function doesn't have any dependencies, this test case can be simplified into:
```
it('should render the SampleAnnotationCtrl for a sample', 
  function() {
    ...
  }
);